runtime/jcl/common/criu.cpp

```diff
@@ -30,6 +30,14 @@ extern "C" {
 
 #if defined(J9VM_OPT_CRIU_SUPPORT)
 
+jlong JNICALL
+Java_openj9_internal_criu_InternalCRIUSupport_getCheckpointRestoreTimeDeltaImpl(JNIEnv *env, jclass unused)
+{
+	PORT_ACCESS_FROM_ENV(env);
+
+	return PORTLIB->checkpointRestoreTimeDelta / 1000000;
```

Comment:
Can we keep the nano seconds. I can see it being useful for MXbeans etc. Perhaps rename this to `Java_openj9_internal_criu_InternalCRIUSupport_getCheckpointRestoreNanoTimeDeltaImpl` and do the division in the caller

Reply:
Sure, renamed.